src/xdocs/config_metrics.xml

```diff
@@ -237,13 +237,14 @@
             <td>excludedClasses</td>
             <td>User-configured class names to ignore</td>
             <td><a href="property_types.html#stringSet">String Set</a></td>
-            <td>IllegalStateException, Set, StringBuilder, HashMap, ArrayList,
-                String, float, SortedSet, long, RuntimeException, NullPointerException,
-                TreeSet, List, Boolean, Void, Queue, Short, IllegalArgumentException,
-                UnsupportedOperationException, HashSet, void, Character, IndexOutOfBoundsException,
-                byte, double, SecurityException, TreeMap, Double, Deque, int, Exception, LinkedList,
-                Integer, Float, StringBuffer, boolean, Byte, SortedMap, char, Long, short,
-                Throwable, Object, Class, ArrayIndexOutOfBoundsException, Map</td>
+            <td>HashMap, ArrayList, String, float, TreeSet, List, Boolean, Void, Override, Short,
```

Comment:
As I mentioned before.
No unrelated refactoring in functions changes.

Please explain change of order and move it in separate commit/PR.

Reply:
I changed order because otherwise I had  `XdocsPagesTest.testAllCheckSections` test failed

Reply:
What was a failure message ?
Why ordering should be like this ?
It worked fine before, what update force whole ordering be different from ordering that we have in master?

Reply:
@romani Ordering in xdoc tests are based on ordering of class fields. This verification doesn't examine the source code so we are wholely dependent on how field specifies the ordering.
`DEFAULT_EXCLUDED_CLASSES` in `AbstractClassCouplingCheck` creates a hashset that is based on the list given to the arrays stream class. Ordering is not being maintained in this conversion.

I don't know the specific rules on how it determines order.
You can see the previous ordering doesn't match either.

> DEFAULT_EXCLUDED_CLASSES
>  "boolean", "byte", "char", "double", "float", "int", ....
> xdoc
> IllegalStateException, Set, StringBuilder, HashMap, ArrayList, ....

Reply:
Ok, thanks a lot for explanation, let's keep a chance in this commit